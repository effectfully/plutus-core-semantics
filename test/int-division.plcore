(program

(module Foo (imported) (exported () ()) (local () ())
  (define i1 (builtin divideInt 30 3))
  (define i2 (builtin divideInt 21 7))
  (define i3 (builtin divideInt 35 3))
  (define i4 (builtin divideInt 20 6)))

)
