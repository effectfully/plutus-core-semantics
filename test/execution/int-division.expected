<T>
  <k> . </k>
  <lenv> .Map </lenv>
  <genv> ( Foo . i3 ) |-> 11 ( Foo . i4 ) |-> 3 ( Foo . i1 ) |-> 10 ( Foo . i2 ) |-> 3 </genv>
  <modules>
    <Module>
      <curr> 0 </curr>
      <name> Foo </name>
      <dataCons> .TyExpList </dataCons>
      <decl> .NameList </decl>
      <importedModules> .ModNmList </importedModules>
      <exportTy> .TyExpList </exportTy>
      <exportTm> .NameList </exportTm>
    </Module>
  </modules>
</T>
